src/axom/slic/interface/slic_macros.hpp

```diff
@@ -62,10 +69,17 @@
     }                                                               \
   } while(axom::slic::detail::false_value)
 
-/// @}
+///@}
 
-/// \name WARNING MACROS
-/// @{
+///@{
+//! \name WARNING MACROS
+//!
+//! \attention These warning macros are collective operations.
+//! All ranks in the user-supplied communicator must call the macro
+//! when used within an MPI distributed environment, and abort on warning
+//! messages is enabled for the current active logger (default is disabled
+//! for loggers)
```

Comment:
This paragraph does need to be here: thank you for adding it.  Could you rephrase it? Currently, the paragraph starts out "These are collective macros."  Later it says "if abort on warning is enabled" and finished up with "default is disabled."